4_pipeline/out/model_RMSE.csv

```diff
@@ -0,0 +1,116 @@
+exper_n,exper_id,model_type,rmse
```

Comment:
It looks like you did add `*/out/*` to your `.gitignore` so that these output files would be ignored, but you may have committed this file accidentally before adding that. The best course of action would be to delete this file through your fork of the code here https://github.com/jortiz-usgs/ds-pipelines-targets-1/tree/makefile/4_pipeline/out